### ISSUE TITLE
Add mouse clicks [Feaute Request]

### ISSUE THREAD

**Issue by ragreenburg** [atx/wtype#24]
It would be nice to add [mouse clicks](https://linuxhint.com/xdotool_stimulate_mouse_clicks_and_keystrokes/) like xdotool has.

**Comment by atx** [comment]
Good idea. Meanwhile, a very barebones implementation can be found in https://github.com/swaywm/wlroots/blob/master/examples/virtual-pointer.c

**Comment by zefir-git** [comment]
this would be extremely useful

**Comment by paboum** [comment]
Whoaaa! It's not a "nice to have", it's a mandatory part of calling something "xdotool for wayland". I'm installing uinput kernel module and will use https://github.com/ReimuNotMoe/ydotool instead.

**Comment by dustymabe** [comment]
@atx WDYT of https://github.com/atx/wtype/pull/56 ?

**Comment by Twig6943** [comment]
This being merged would be really cool
